src/network/http/websocket.cpp

```diff
@@ -188,6 +165,23 @@ namespace fc { namespace http {
               return _ws_connection->get_request_header(key);
             }
 
+            /****
+             * @brief retrieves the remote hostname
+             * 
+             * @param forward_header_key the key to look at in the request header
+             * @returns the value in the header, otherwise the remote endpoint
+             */
+            virtual std::string get_remote_hostname(const std::string& forward_header_key)
```

Comment:
Method name is misleading. This doesn't get the hostname but the IP address.

Reply:
I agree. I changed it to get_remote_ip.